README.md

```diff
@@ -1,17 +1,54 @@
 # Python Community News
+![PCN Logo](.github/assets/PCN%20Logo%20V0.16.jpg)
 
-
-### Watch the stream:
-
+- Check out <https://pythoncommunitynews.com>
 - [Watch the Show on Twitch](https://twitch.tv/kjaymiller)
 - [What the Shown on YouTube](https://youtube.com/c/kjaymiller)
+- https://buttondown.email/python-community-news
+## Topics
 
+The Python
 
-### Subscribe to the Newsletter:
+> Non-Pipable: Any news or topic that do not result in someone needed to `pip install` something.
 
-- https://buttondown.email/python-community-news
+#### Examples of great topics:
+
+- Updates from the [Python Software Foundation (PSF)](https://www.python.org/psf/)
+- Posts from [Pythonistas][^1] that address a concept or thought that could be helpful to other Pythonistas.
+- General Tech News or Topics that, while not necessarily related to the Python programming language, are still important to Pythonistas.
+- Topics that related to an underrepresented group of Pythonistas.
+- Topics that focus/showcase efforts Python communmities that are often overlooked.
+
+> IMPORTANT: We recognize that the experiences of marginalized voices aren't always a shared by the majority of folks. We want to ensure that we cover topics with that understanding and hope our coverage of topics will be done with that in mind. We also want to ensure that in areas that we cannot cover, we will do our best to provide a perspective that can as well as be transparent of any conflicts of interest that may arise.
+
+### Add a Topic
+
+> :white_check_mark: Review the [Code of Conduct](CODE_OF_CONDUCT.md) before adding a topic or issue in general.
+
+You can add some things you want us to talk about by creating an 
+[issue](https://github.com/kjaymiller/Python-Community-News/issues/new/choose).
+
+![Add a Topic](.github/assets/Add%20Topic.gif)
+## Conference Lightning Round
+
+We also do a Conference Lightning Round to create shorter than lightning talk coverage of upcoming CFPs and Conferences. You can also view our existing list of Upcoming CFPs and Conferences in our [Python Community News Project Board](https://github.com/users/kjaymiller/projects/4/views/3).
+
+We cover conferences and CFPs that are focused on the following topics:
+
+- Python (General)
+- Data Science Where Python Is One of Dominant Languages
+- A Python Package
+- A Python Web Framework(s)
+- Python in Relation to a Specific Category or Topic
```

Comment:
My general rule for this kind of broad categorization (and to hedge against missing something important when enumerating topics) is
> Anything that would be of interest to the Python community

Reply:
I had something similar to this but took it out as there are many conferences that are general tech events that someone interested in python would be interested with (Looking at All things Open, DevOps Days). That I think would qualify but I don't know if we want to include every tech conference. maybe we do and this can be something that we discuss more in detail.